site/src/modules/management/ManagementSettingsLayout.tsx

```diff
@@ -1,28 +1,27 @@
 import type { DeploymentConfig } from "api/api";
 import { deploymentConfig } from "api/queries/deployment";
-import type { AuthorizationResponse, Organization } from "api/typesGenerated";
+import type { AuthorizationResponse } from "api/typesGenerated";
 import { ErrorAlert } from "components/Alert/ErrorAlert";
 import { Loader } from "components/Loader/Loader";
 import { Margins } from "components/Margins/Margins";
 import { Stack } from "components/Stack/Stack";
 import { useAuthenticated } from "contexts/auth/RequireAuth";
 import { RequirePermission } from "contexts/auth/RequirePermission";
+import type { Permissions } from "contexts/auth/permissions";
 import { useDashboard } from "modules/dashboard/useDashboard";
 import { type FC, Suspense, createContext, useContext } from "react";
 import { useQuery } from "react-query";
-import { Outlet, useParams } from "react-router-dom";
+import { Outlet, useLocation } from "react-router-dom";
 import { Sidebar } from "./Sidebar";
 
+type ManagementSettingsValue = Readonly<{
+	deploymentValues: DeploymentConfig | undefined;
+}>;
```

Comment:
Removed most of the properties for two reasons:
1. The `Sidebar` component depended on the context provider. So it was impossible to show the sidebar at all times, because if you moved it outside the provider, it would blow up
2. It's duplicate state that was already accessible via other hooks. Felt better to beef those up, because then other pages would be able to benefit, too